### PR TITLE
integrate and open diversity form

### PR DESCRIPTION
**Pull request by skorbut**
This adds the link to the diversity form and the start and end date.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -124,7 +124,7 @@ <h3>## Venue</h3>
         <a href="https://duckduckgo.com/?q=Finkenau+35%2C+22081+Hamburg&iaxm=maps&atb=v116-1&ia=maps">
           <h4>HAW Hamburg</h4>
           <h4>Campus Finkenau</h4>
-      
+
```
Comment (emrox) [inline index.html]: unrelated?

File: index.html [hunk 2]
```diff
@@ -199,7 +199,7 @@ <h3>## Diversity</h3>
         <br />
 
         <p>
-          We'll inform you as soon as the application phase for diversity tickets will begin.
+          The application phase started shortly after the ticket sale. Use the <a target= "_blank" href = "https://docs.google.com/forms/d/e/1FAIpQLSdRY_j_snQky14Szoe9M3CztqQ4_AJcaovt8ebF3Px2a8-H_g/viewform">diversity form</a> to apply. The application phase will end at 2020/03/27 and applicants will be contacted before 2020/04/01.
```
Comment (emrox) [inline index.html]: please use shorter lines

Comment (emrox) [inline index.html]: maybe just `The application phase is live. Use the ...` ?

File: index.html (hunk 1 above)
Comment (skorbut) [reply index.html]: indeed

File: index.html (hunk 2 above)
Comment (skorbut) [reply index.html]: or has started?

Comment (skorbut) [reply index.html]: The link to the form is quite long.

File: index.html [hunk 3]
```diff
@@ -199,7 +198,8 @@ <h3>## Diversity</h3>
         <br />
 
         <p>
-          We'll inform you as soon as the application phase for diversity tickets will begin.
+          The application phase has started. Use the <a target= "_blank" href = "https://docs.google.com/forms/d/e/1FAIpQLSdRY_j_snQky14Szoe9M3CztqQ4_AJcaovt8ebF3Px2a8-H_g/viewform">diversity form</a>
```
Comment (emrox) [inline index.html]: you can add a line break before the link

```suggestion
          The application phase has started. Use the
          <a target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSdRY_j_snQky14Szoe9M3CztqQ4_AJcaovt8ebF3Px2a8-H_g/viewform">diversity form</a>
```

also note there should be no spaces before & after the `=` sign on HTML attributes

**Review by emrox — APPROVED** [review]
